From 868f0d36979c6930efcc8be20ed355cc1205723c Mon Sep 17 00:00:00 2001
From: Chiitoo <Chiitoo@users.noreply.github.com>
Date: Sat, 21 Nov 2020 03:52:22 +0200
Subject: [PATCH] Fix compiling with Qt >=5.15.3 (#588)

After a Qt upstream change [1], certain assignments that were not
caught by the compiler will now fail hard due to 'use of deleted
function'.

1. https://code.qt.io/cgit/qt/qtbase.git/commit/?id=f8ad329f7ee
---
 src/filedialog.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/filedialog.cpp b/src/filedialog.cpp
index 51754ce0..ef28696e 100644
--- a/src/filedialog.cpp
+++ b/src/filedialog.cpp
@@ -1109,7 +1109,7 @@ void FileDialog::setMimeTypeFilters(const QStringList& filters) {
                 nameFilter += suffix;
                 nameFilter += QLatin1Char(' ');
             }
-            nameFilter[nameFilter.length() - 1] = ')';
+            nameFilter[nameFilter.length() - 1] = QLatin1Char(')');
         }
         nameFilters << nameFilter;
     }
